WordPress/gutenberg
# Navigation: adjust mobile close button position when admin bar is visible

## What?

This PR adjusts the position of the close button in the mobile navigation dialog when the user is logged in (when the admin bar is visible). Before, the close button would seem to jump in position compared to the hamburger menu.

## Why?

This isn't an issue when the admin bar is not visible, so this aligns the two views.

## How?

I've added a top margin to the dialog div which matches the `$admin-bar-height` or `$admin-bar-height-big`, depending on the breakpoint.

## Testing Instructions




1. Activate Twenty Twenty-Two
2. Make sure there is a navigation block in the header with the Overlay Menu set to Mobile
```



```
3. Make sure you're logged in
4. On the front end, adjust your screen size to around 440px wide
5. Open the mobile menu dialog by clicking the hamburger menu
6. You should see the close button position is in the same position as the hamburger menu icon

## Screenshots or screencast 

Before:
<video src="https://user-images.githubusercontent.com/1645628/173099015-01592d14-2d10-437a-b3ac-9f7dc89aded7.mov" width=180 />

After:
<video src="https://user-images.githubusercontent.com/1645628/173098981-512c50f0-2f6b-4e24-a4a0-2b099bad8c71.mov" width=180 />
